src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/ReceivedMessagesHealthControllerTest.java

```diff
@@ -191,6 +198,57 @@ void test_health_reports_success_if_messages_check_disabled_in_current_environme
         verifyNoInteractions(caseEventMessageRepository);
     }
 
+    @ParameterizedTest
+    @MethodSource(value = "enabledEnvProvider")
+    void test_enabled_for_environment_return_false_prod_aat(String env) {
+
+        setField(receivedMessagesHealthController, "receivedMessageCheckEnvEnabled", "prod,aat");
+
+        MockHttpServletRequest mockHttpServletRequest = new MockHttpServletRequest();
+        mockHttpServletRequest.setServerName("case-event-handler.any.environment");
+        mockHttpServletRequest.setRequestURI("/health");
+
+        RequestContextHolder.setRequestAttributes(new ServletRequestAttributes(mockHttpServletRequest));
+
+        assertFalse(receivedMessagesHealthController.isNotEnabledForEnvironment(env));
+    }
+
+    @ParameterizedTest
+    @MethodSource(value = "disabledEnvProvider")
+    void test_enabled_for_environment_return_true_demo_ithc(String env) {
```

Comment:
Test name contradicts test purpose. Should the name be test_not_enabled_for_environment_return_true_demo_ithc?